pkg/apis/serving/v1alpha1/revision_validation.go

```diff
@@ -198,6 +198,11 @@ func validateContainerPorts(ports []corev1.ContainerPort) *apis.FieldError {
 		errs = errs.Also(apis.ErrDisallowedFields(disallowedFields...))
 	}
 
+	// Don't allow userPort to conflict with QueueProxy sidecar
+	if userPort.ContainerPort == 8012 || userPort.ContainerPort == 8022 {
```

Comment:
We should be able to use the defined constants instead of hardcoded numbers for these.

Reply:
constants are here

https://github.com/knative/serving/blob/master/pkg/queue/constants.go

Reply:
+1

Reply:
I needed to move the constants to prevent a go import cycle. I moved to v1alpha1 with the UserPort constants as it seemed the most appropriate place without creating a new place.